app/app/urls.py

```diff
@@ -143,12 +151,6 @@
     url(r'^tools/?', dashboard.views.toolbox, name='tools'),
     url(r'^gas/?', dashboard.views.gas, name='gas'),
 
-    # redeem coin
```

Comment:
whats fun is that we could now programatically create tips and use those to redeem coin airdrops

Reply:
or even extend the current receive flow to accept 721s